### ISSUE TITLE
Can jquery be removed as core dependency in close future?

### ISSUE THREAD

**Issue by dev7ch** [yiisoft/yii2#16147]
### What steps will reproduce the problem?
Setting up a Yii application

### What is the expected result?

Pure javascript (es6)

### What do you get instead?

JQuery is implemented in the core

### Additional info

As @klimov-paul wrote in #12588 JQuery is used only on js asset option `View::POS_HEAD` and `View::POS_READY`.

Could this simply be replaced by klimov-paul mention?

```
document.addEventListener("DOMContentLoaded", function(event) { 
  //do work
});
```
or the basic idea of 

```
window.addEventListener('load', function () {
//your code right here;
}, false);
```

Image you build a modern frontendend application based on e.g. Vue/Nuxt or sth like Google AMP, jQuery isn`t needed and in fact there is an unused assets without function.

### Question

Are there plans to drop jQuery as Core dependency? in the next months/year or v3.0?

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.?
| PHP version      | 7.2
| Operating system | UNIX

**Comment by rob006** [comment]
This is already fixed in 2.1 branch.

**Comment by rustamwin** [comment]
Use your config file:
```
'assetManager'       => [
            'linkAssets'      => true,
            'appendTimestamp' => true,
            'bundles' => [
                \yii\web\JqueryAsset::class => false //or setting up path to your jquery file
            ]
        ],
```
